config

```diff
@@ -6,5 +6,5 @@
 # in the file LICENSE in the source distribution or at
 # https://www.openssl.org/source/license.html
 
-THERE=`dirname $0`
+THERE=$(dirname $0)
```

Comment:
Interesting question whether this should be done, or the greater portabality of backtick should be preserved.

Reply:
Solaris 10 default shell *might* have an issue with \`...\` vs $(...) but I have no way of testing that.